client/trino-jdbc/src/test/java/io/trino/jdbc/TestJdbcConnection.java

```diff
@@ -495,7 +495,7 @@ public void testConcurrentCancellationOnStatementClose()
             // verify that the query was cancelled
             assertThatThrownBy(future::get).isNotNull();
             assertThat(listQueryErrorCodes(sql))
-                    .containsExactly("USER_CANCELED")
+                    .containsOnly("TRANSACTION_ALREADY_ABORTED", "USER_CANCELED");
```

Comment:
will the assertion pass when TRANSACTION_ALREADY_ABORTED is not found?

Reply:
Will the size be only 1 here ? Should we remove this ?

Reply:
> Will the size be only 1 here ?

That is the plan, however the actual error code as flaky test proved may vary.